apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/context/AbstractTracerContext.java

```diff
@@ -138,4 +138,10 @@ public interface AbstractTracerContext {
      * Get current primary endpoint name
      */
     String getPrimaryEndpointName();
+
+    /**
+     *  Change the current context to IgnoredTracerContext.
```

Comment:
```suggestion
     *  Change the current context to be in ignoring status.
```

It doesn't have to be `IgnoredTracerContext`. The only effect is to make the spans not being reported anymore.